Auth0Tests/OAuth2GrantSpec.swift

```diff
@@ -221,11 +215,25 @@ class OAuth2GrantSpec: QuickSpec {
                 }
             }
 
-            it("shoud fail credentials no nonce") {
-                pkce = PKCE(authentication: authentication, redirectURL: redirectURL, verifier: verifier, challenge: challenge, method: method, responseType: response)
+            it("should fail with no nonce") {
```

Comment:
What does not have a nonce, the token or something else? The token here is going to come back as malformed. Is there a specific error or error message you could look for to make sure it's failing the way you expect?

Reply:
The validator would not have a nonce to compare, in this case. Though the token should be a valid one. Thanks, will rephrase it and add a valid token.

Reply:
Fixed in ca28451be12118c8928745aead45feaa63e669e1.